### PR TITLE
Tweaks to rich text editor

### PR DESCRIPTION
**Pull request by Anahkiasen**
### Changed

- Use soft newlines when pressing "Enter"
- Make editor as tall as previous description field
- Make editor resizable in height

### Removed

- Removed "underline" option

---

Ticket: https://jira.uitdatabank.be/browse/III-5115

### CODE REVIEW

File: src/pages/steps/AdditionalInformationStep/DescriptionStep.tsx [hunk 1]
```diff
@@ -159,6 +165,33 @@ const DescriptionStep = ({
...
+        block.getInlineStyleAt(newSelection.getStartOffset()),
+        null,
+      );
+
+      setEditorState(
+        EditorState.push(editorState, newContent, 'insert-fragment'),
+      );
+    }
+
+    return true;
```
Comment (brampauwelyn) [inline src/pages/steps/AdditionalInformationStep/DescriptionStep.tsx:192]: Is it necessary to add the  `return true` here? 
```suggestion
```

File: src/pages/steps/AdditionalInformationStep/DescriptionStep.tsx [hunk 2]
```diff
@@ -159,6 +165,33 @@ const DescriptionStep = ({
...
+        '\n',
+        block.getInlineStyleAt(newSelection.getStartOffset()),
+        null,
+      );
+
+      setEditorState(
+        EditorState.push(editorState, newContent, 'insert-fragment'),
+      );
+    }
+
```
Comment (brampauwelyn) [inline src/pages/steps/AdditionalInformationStep/DescriptionStep.tsx:191]: Replace `if else` structure with if and early return
```suggestion
    if (selection.isCollapsed()) {
      setEditorState(RichUtils.insertSoftNewline(editorState));
      return
    } 
      const content = editorState.getCurrentContent();
      let newContent = Modifier.removeRange(content, selection, 'forward');
      const newSelection = newContent.getSelectionAfter();
      const block = newContent.getBlockForKey(newSelection.getStartKey());

      newContent = Modifier.insertText(
        newContent,
        newSelection,
        '\n',
        block.getInlineStyleAt(newSelection.getStartOffset()),
        null,
      );

      setEditorState(
        EditorState.push(editorState, newContent, 'insert-fragment'),
      );
 

```

**Review by brampauwelyn — APPROVED** [review]

File: src/pages/steps/AdditionalInformationStep/DescriptionStep.tsx (hunk 1 above)
Comment (Anahkiasen) [reply src/pages/steps/AdditionalInformationStep/DescriptionStep.tsx:192]: It has special meaning in Draft to indicate whether the newline was "handled" or whether the native behavior needs to kick in in addition to the callback. It's basically an `event.preventDefault()`

Comment (brampauwelyn) [reply src/pages/steps/AdditionalInformationStep/DescriptionStep.tsx:192]: Ok clear

**Review by simon-debruijn — APPROVED** [review]

**Pull request merged**